schema-registry/src/test/scala/net/manub/embeddedkafka/schemaregistry/streams/ExampleKafkaStreamsSchemaRegistrySpec.scala

```diff
@@ -1,24 +1,27 @@
-package net.manub.embeddedkafka.streams
+package net.manub.embeddedkafka.schemaregistry.streams
 
-import net.manub.embeddedkafka.avro.schemaregistry._
-import net.manub.embeddedkafka.avro.schemaregistry.Codecs._
 import net.manub.embeddedkafka.Codecs._
 import net.manub.embeddedkafka.ConsumerExtensions._
-import net.manub.embeddedkafka.{EmbeddedKafkaConfig, TestAvroClass}
-import org.apache.kafka.common.serialization.{Serde, Serdes}
+import net.manub.embeddedkafka.TestAvroClass
+import net.manub.embeddedkafka.schemaregistry.avro.Codecs._
+import net.manub.embeddedkafka.schemaregistry.{
+  EmbeddedKafkaConfigWithSchemaRegistry,
+  _
+}
+import org.apache.kafka.common.serialization._
 import org.apache.kafka.streams.kstream.{KStream, Produced}
 import org.apache.kafka.streams.{Consumed, StreamsBuilder}
 import org.scalatest.{Matchers, WordSpec}
 
 class ExampleKafkaStreamsSchemaRegistrySpec
```

Comment:
Could we drop the "SchemaRegistry" part since this class is already on a different package?